### PR TITLE
resolve open bounties background disappear on hover issue 

### PR DESCRIPTION
**Pull request by MahtabBukhari**
## Issue ticket number and link
- Issue ticket number: #60
- [Link](https://github.com/stakwork/sphinx-tribes-frontend/issues/60)

## Evidance 
- Evidance video [Link](https://www.loom.com/share/19f3c423f5ac407c8579099222ce9fa2?sid=cefde0a7-2a64-4225-8f37-7ab1e9957af1)

## Changes
- AssignedUnassignedBounties.tsx
1. The pull request includes styling adjustments for the `Bounties` React component.
2. Notable changes involve border styling in the `DescriptionPriceContainer` and adjustments in `UnassignedPersonProfile`.
3. `BountyContainer` now includes `box-sizing: border-box;` and updated styles for `:hover`.
4. `BountyLink` and `BountyContainer` (in the else block) have new `style` attributes setting width and height.

- AssignedUnassignedBounties.spec.tsx
1. The updated code introduces a new test case for the `Bounties` component, focusing on the hover effect of the `DescriptionPriceContainer`.
2. The `onPanelClick` function in `bountyProps` is now created using `jest.fn()` for better mock functionality.
3. Event simulation with `fireEvent.mouseEnter` is employed to mimic a mouse enter event on the `DescriptionPriceContainer`.
4. The test verifies that the container has the expected style (`border: 2px solid rgba(0,130,255,0.2)`) after the mouse enter event.
5. The test case enhances coverage by specifically checking the hover effect on the designated container in the `Bounties` component.

### CODE REVIEW

**Comment by MahtabBukhari** [comment]
@ecurrencyhodler @elraphty please review my code now i have solved branch conflicts

**Comment by aliraza556** [comment]
Hi @MahtabBukhari,
Please close this PR https://github.com/stakwork/sphinx-tribes-frontend/pull/82.

**Comment by aliraza556** [comment]
> > Hi @MahtabBukhari, Please close this PR #82.
> 
> @aliraza556 Could you help me to how close it?

It simple just click [here](https://i.imgur.com/OZH1o1o.png)

**Comment by MahtabBukhari** [comment]
> > > Hi @MahtabBukhari, Please close this PR #82.
> > 
> > 
> > @aliraza556 Could you help me to how close it?
> 
> It's simple, just click [here](https://i.imgur.com/OZH1o1o.png)

@aliraza556 thank you so much

**Comment by aliraza556** [comment]
> thank you so much

You're welcome! If there's anything else you need help with, feel free to ask. Just one more thing, there's no need to create a new PR; simply remove the conflicts and commit the changes.

File: src/people/utils/AssignedUnassignedBounties.tsx [hunk 1]
```diff
@@ -204,10 +199,11 @@ const Bounties = (props: BountiesProps) => {
         </BountyLink>
       ) : (
         <BountyContainer color={color}>
-          <DescriptionPriceContainer unAssignedBackgroundImage='url("/static/unassigned_bounty_bg.svg")'>
-            <BountyLink
-              to={props.org_uuid ? `/bounty/${props.id}/${props.org_uuid}` : `/bounty/${props.id}`}
```
Comment (kevkevinpal) [inline src/people/utils/AssignedUnassignedBounties.tsx:209]: why is this `to` value modified?

File: src/people/utils/AssignedUnassignedBounties.tsx [hunk 2]
```diff
@@ -156,9 +153,7 @@ const Bounties = (props: BountiesProps) => {
   return (
     <>
       {!!assignee?.owner_pubkey && !!assignee?.owner_alias ? (
-        <BountyLink
-          to={props.org_uuid ? `/bounty/${props.id}/${props.org_uuid}` : `/bounty/${props.id}`}
```
Comment (kevkevinpal) [inline src/people/utils/AssignedUnassignedBounties.tsx:160]: looks like this `to` value was modified in this change

Comment (MahtabBukhari) [reply src/people/utils/AssignedUnassignedBounties.tsx:160]: I am going to resolve this one just give me few minutes

**Comment by MahtabBukhari** [comment]
@kevkevinpal I haved fixed it please review it now

**Comment by MahtabBukhari** [comment]
@kevkevinpal I have resolved conflict issues that are due to frequently updating master branch. Could you review now?

**Comment by ecurrencyhodler** [comment]
Tested. Good work. Paid.

**Pull request merged**